travis-scripts/before_deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ev
+echo "Starting travis bash script: before_deploy.sh"
+# A tag ending with "number.number" is not a prerelease
+if echo $DIST_TARBALL | grep -E -q '[0-9]{1,2}\.[0-9]{1,2}\.tar\.gz';
+then
+    IS_PRERELEASE=false;
+else
+    IS_PRERELEASE=true;
+fi;
+export IS_PRERELEASE
```

Comment:
Exporting a variable means that the *children* of this process get it, not the parent. So the parent travis script will not see IS_PRERELEASE. We need to figure out a different way for this.